tests/unit/auth/v4/constructStringToSign.js

```diff
@@ -7,119 +7,132 @@ const constructStringToSign =
 const DummyRequestLogger = require('../../helpers').DummyRequestLogger;
 
 const log = new DummyRequestLogger();
-
-describe('constructStringToSign function', () => {
-    // Example taken from: http://docs.aws.amazon.com/AmazonS3/
-    // latest/API/sig-v4-header-based-auth.html
-    it('should construct a stringToSign in accordance ' +
-        'with AWS rules for a get object request (header auth)', () => {
-        const params = {
-            request: {
-                method: 'GET',
-                path: '/test.txt',
-                headers: {
-                    'host': 'examplebucket.s3.amazonaws.com',
-                    'x-amz-date': '20130524T000000Z',
-                    'authorization': 'AWS4-HMAC-SHA256 Credential' +
-                        '=AKIAIOSFODNN7EXAMPLE/20130524/us-east-1/' +
-                        's3/aws4_request,SignedHeaders=host;range;' +
-                        'x-amz-content-sha256;x-amz-date,Signature=' +
-                        'f0e8bdb87c964420e857bd35b5d6ed310bd44f' +
-                        '0170aba48dd91039c6036bdb41',
-                    'range': 'bytes=0-9',
-                    'x-amz-content-sha256': 'e3b0c44298fc1c149afbf4c' +
-                        '8996fb92427ae41e4649b934ca495991b7852b855',
+[
+    { path: '', desc: 'constructStringToSign function' },
+    { path: '/_/proxy', proxyPath: undefined,
```

Comment:
I think `proxyPath` is not used anywhere (or maybe you could rename `path` to `proxyPath`)